torch/csrc/jit/codegen/cuda/graph_fuser.cpp

```diff
@@ -1276,6 +1292,48 @@ void PeepholeOptimizeShapeExpressions(Block* block) {
   }
 }
 
+Value* guardView(
+    Node* fusion,
+    std::unordered_map<Value*, Value*>& valueToSize,
+    Node* versioning_if,
+    Node* view,
+    Value* view_sizes_profile_ivalue) {
```

Comment:
nitpick: This name is a little confusing here. Can we comment that it's the runtime argument of view size, instead of the constant int list recorded during profiling runs